pallets/parachain-utils/src/lib.rs

```diff
@@ -109,27 +109,46 @@ impl<AccountId> User<AccountId> {
 
 #[derive(Encode, Decode, RuntimeDebug)]
 pub enum Error {
+    /// Account is blocked in a given space.
+    AccountIsBlocked,
+    /// Content is blocked in a given space.
+    ContentIsBlocked,
     /// IPFS CID is invalid.
     InvalidIpfsCid,
     /// `Raw` content type is not yet supported.
     RawContentTypeNotSupported,
     /// `Hyper` content type is not yet supported.
     HypercoreContentTypeNotSupported,
+    /// Space handle is too short.
+    HandleIsTooShort,
+    /// Space handle is too long.
+    HandleIsTooLong,
+    /// Space handle contains invalid characters.
+    HandleContainsInvalidChars,
     /// Content type is `None`.
     ContentIsEmpty,
 }
 
 impl From<Error> for &'static str {
     fn from(e: Error) -> &'static str {
         match e {
+            Error::AccountIsBlocked => "AccountIsBlocked",
+            Error::ContentIsBlocked => "ContentIsBlocked",
             Error::InvalidIpfsCid => "InvalidIpfsCid",
             Error::RawContentTypeNotSupported => "RawContentTypeNotSupported",
             Error::HypercoreContentTypeNotSupported => "HypercoreContentTypeNotSupported",
+            Error::HandleIsTooShort => "HandleIsTooShort",
+            Error::HandleIsTooLong => "HandleIsTooLong",
+            Error::HandleContainsInvalidChars => "HandleContainsInvalidChars",
             Error::ContentIsEmpty => "ContentIsEmpty",
         }
     }
 }
```

Comment:
Idea:
We can use [`IntoStaticStr`](https://docs.rs/strum_macros/0.24.0/strum_macros/derive.IntoStaticStr.html) from [`strum_macros`](https://crates.io/crates/strum_macros) package so this is more maintainable.